.gitignore

```diff
@@ -39,3 +39,4 @@ package-lock.json
 .java-version
 
 .env
+/nbproject/
```

Comment:
Was this accidental?  Or is this something we want added in .gitignore?  (This might be from NetBeans, not sure...if it is NetBeans files then I'm OK with leaving this here. Just wanted to ask)

Reply:
Yes, that's from NetBeans.  At the moment I seem to be the only contributor using NB with dspace-angular.  I noticed this but decided to let it through.  It should be harmless, and I didn't want to slip some day and include local NB book-keeping in a commit.